### ISSUE TITLE
Yarn create nuxt app  failed to create the project, and the yarn command is missing command not found:yarn

### ISSUE THREAD

**Issue by sdark1326632** [nuxt/create-nuxt-app#676]
Yarn create nuxt app < my project > failed to create the project, and the yarn command is missing command not found:yarn

System: Mac os high sierra 10.13.6
Node: v14.15
yarn: homebrew install v1.22.10

**Comment by danielroe** [comment]
@sdark1326632 Can you clarify what command you've run? (It should be: `yarn create nuxt-app myproject`.)

**Comment by sdark1326632** [comment]
> @sdark1326632 Can you clarify what command you've run? (It should be: `yarn create nuxt-app myproject`.)

Yes, execute the command yarn create nuxt app, and then fail, and then yarn is lost

**Comment by danielroe** [comment]
@sdark1326632 Those are different commands. Note the `-` character.

**Comment by sdark1326632** [comment]
> @sdark1326632 Those are different commands. Note the `-` character.

Yes, there are. It can't be reproduced today. I'll take a screenshot when it appears
